Source/WebCore/html/CanvasBase.h

```diff
@@ -95,7 +95,7 @@ class CanvasBase {
 
     virtual AffineTransform baseTransform() const;
 
-    void makeRenderingResultsAvailable(ShouldApplyPostProcessingToDirtyRect = ShouldApplyPostProcessingToDirtyRect::Yes);
+    RefPtr<ImageBuffer> makeRenderingResultsAvailable(ShouldApplyPostProcessingToDirtyRect = ShouldApplyPostProcessingToDirtyRect::Yes);
```

Comment:
Is it possible to make `buffer()` private? Or renamed to something that makes it less tempting to use?

Reply:
Yeah, working on it but in some future patch.